html/changelog.html

```diff
@@ -23,26 +23,27 @@
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<div align='center'><font size='3'><b>Traditional Games Space Station 13</b></font></div>
-			
-			<p><div align='center'><font size='3'><a href="http://www.tgstation13.org/phpBB/">Forum</a> | <a href="http://www.tgstation13.org/wiki/Main_Page">Wiki</a> | <a href="https://github.com/tgstation/-tg-station">Source</a></font></div></p>
-			<font size='2'><b>Visit our IRC channel:</b> #tgstation13 on irc.rizon.net</font>
+			<div align='center'><font size='3'><b>Citadel Station</b></font></div>
+			<div align='center'><font size='3'><b>A downstream of /tg/Station </b></font></div>
+			<p><div align='center'><font size='3'><a href="http://www.tgstation13.org/phpBB/">TG's Forum</a> | <a href="http://www.tgstation13.org/wiki/Main_Page">Wiki</a> | <a href="https://github.com/tgstation/-tg-station">Source</a></font></div></p>
+			<font size='2'><b>Visit tg's IRC channel:</b> #tgstation13 on irc.rizon.net</font>
+			<font size='3'><a href="https://discord.gg/3gJ9pnM"><b>...Or our Discord!</b></a></font>
 			</td>
 	</tr>
 </table>
 
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<font size='2'><b>Current Project Maintainers:</b> <a href='https://github.com/tgstation?tab=members'>-Click Here-</a><br></font>
-			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/tgstation/-tg-station/graphs/contributors'>-Click Here-</a><br></font>

```

Comment:
Not really anything worth noting but I'm laughing pretty hard that TG's changelog has had outdated links for so long